python/lsst/ts/observatory/control/maintel/mtcs.py

```diff
@@ -1928,6 +1928,42 @@ async def get_m1m3_applied_balance_forces(self) -> salobj.type_hints.BaseMsgType
             )
         )
 
+    async def get_m1m3_slew_controller_settings(self) -> dict:
+        """
+        Retrieve the current M1M3 slew controller settings.
+
+        Returns
+        -------
+        dict
+            A dictionary containing the current settings, where the keys are
+            the names used in the SetSlewControllerSettings enumeration.
+
+        Raises
+        ------
+        RuntimeError
+            If the expected attribute is not found in the event.
+        """
+        settings_event = await self.rem.mtm1m3.evt_slewControllerSettings.aget(
+            timeout=self.fast_timeout
+        )
+
+        expected_attributes = {
+            "ACCELERATIONFORCES": "useAccelerationForces",
+            "BALANCEFORCES": "useBalanceForces",
+            "BOOSTERVALVES": "triggerBoosterValves",
+            "VELOCITYFORCES": "useVelocityForces",
+        }
+
+        settings = {}
+        for key, attr in expected_attributes.items():
+            if not hasattr(settings_event, attr):
+                raise RuntimeError(
+                    f"Expected attribute '{attr}' not found in slew controller settings event."
```

Comment:
Maybe print what are the valid ones? something like. 

f"Expected attribute '{attr}' not found in slew controller settings event. "
f"Must be one of {expected_attributes.keys()}."